### ISSUE TITLE
logs without level about "retransmission failure"

### ISSUE THREAD

**Issue by AskAlexSharov** [anacrolix/torrent#720]
version 1.40.0
```
sctp ERROR: 2022/02/02 15:41:22 [0xc102917c00] retransmission failure: T1-init
sctp ERROR: 2022/02/02 15:41:22 [0xc141a968c0] retransmission failure: T1-cookie
sctp ERROR: 2022/02/02 15:41:22 [0xc05921e380] retransmission failure: T1-init
```

**Comment by anacrolix** [comment]
Good find. What do you think about me implementing the webrtc logging interface to tap into anacrolix/torrent.Client's logger? Looks like they've exposed it quite nicely. I wonder if this error in general is something that needs to be reported upstream.

**Comment by anacrolix** [comment]
https://gophers.slack.com/archives/CAK2124AG/p1643854599797609

**Comment by AskAlexSharov** [comment]
Good for me, thank you.

**Comment by anacrolix** [comment]
Fixed by #735.
